services/airflow/src/lib/dag_helpers/load_dataframe_from_postgres.py

```diff
@@ -0,0 +1,8 @@
+import ramda as R
+import pandas as pd
+from database import DBContext
+
+
+@R.curry
+def execute_query_and_return_dataframe(query: str, context: DBContext):
+    return pd.read_sql(query, con=context["connection"])
```

Comment:
I extracted this from an old dag and wrote some tests, because I want to use use it in the new dag too.